plugin-server/src/main/ingestion-queues/session-recording-v2/consumer.ts

```diff
@@ -112,19 +114,20 @@ export class SessionRecordingIngester {
 
         const offsetManager = new KafkaOffsetManager(this.commitOffsets.bind(this), this.topic)
         const metadataStore = new SessionMetadataStore(producer)
-        const fileStorage = s3Client
+        this.fileStorage = s3Client
             ? new S3SessionBatchFileStorage(
```

Comment:
qq - why not just pass in the config here? Feels like a lot of your classes take hyper specific options which just makes for a lot of verbose code rather than just referencing the config? Could still use Pick<> to only get the values you want but makes for less churn